code/obj/decal/cleanable.dm

```diff
@@ -7,9 +7,9 @@
 ////////////////
 
 ////////////////
-proc/make_cleanable(var/type,var/loc,var/list/viral_list)
+proc/make_cleanable(var/type,var/loc)
 	RETURN_TYPE(/obj/decal/cleanable)
-	return new type(loc, viral_list)
+	return new type(loc)
```

Comment:
Isn't this proc just a `new` wrapper now (ie completely unnecessary). Out of scope for this PR but should be noted!!!